docs/_includes/expressive-loader.html

```diff
@@ -142,11 +142,15 @@ <h3 id="expressive-loader-messages-no-reduced-motion">
   </div>
   {% endhighlight %}
 
-  <h4>
-    Multiple messages (default behavior)
-  </h4>
+  <h3 id="expressive-loader-multiple-messages">
+    Expressive Loader with Multiple Messages
+  </h3>
+  <p>
+    For the following examples, we're using Javascript to rotate through messages.
```

Comment:
```suggestion
    For the following examples, we're using Javascript to rotate through messages. This interaction is not included in Skin, but is only added to these docs. To use this in a similar fashion when using Skin as a static library, you would need to add in that Javascript yourself.
```

Let's try to be extremely clear and explicit about how this works and how it should be used.